DnaComServer/DnaComServer.csproj

```diff
@@ -67,19 +72,18 @@ del "$(TargetDir)ExcelDna.Integration.dll"
 
 REM Re-run the packing to include the .tlb inside the packed files for distribution
 REM Note: Might need to change depending on where packages directory is
-"$(ProjectDir)\packages\ExcelDna.AddIn.1.1.1\tools\ExcelDnaPack.exe" "$(ProjectDir)$(OutDir)$(TargetName)-AddIn.dna" /Y  /O "$(ProjectDir)$(OutDir)$(TargetName)-AddIn-packed.xll"
-"$(ProjectDir)\packages\ExcelDna.AddIn.1.1.1\tools\ExcelDnaPack.exe" "$(ProjectDir)$(OutDir)$(TargetName)-AddIn64.dna" /Y  /O "$(ProjectDir)$(OutDir)$(TargetName)-AddIn64-packed.xll"
+"$(ProjectDir)\packages\ExcelDna.AddIn.1.5.1\tools\ExcelDnaPack.exe" "$(ProjectDir)$(OutDir)$(TargetName)-AddIn.dna" /Y  /O "$(ProjectDir)$(OutDir)$(TargetName)-AddIn-packed.xll"
+"$(ProjectDir)\packages\ExcelDna.AddIn.1.5.1\tools\ExcelDnaPack.exe" "$(ProjectDir)$(OutDir)$(TargetName)-AddIn64.dna" /Y  /O "$(ProjectDir)$(OutDir)$(TargetName)-AddIn64-packed.xll"
 
 REM Register COM servers in add-in on this machine for testing
 REM Note: Change this to -AddIn64.xll if the 64-bit version of Excel is installed
-regsvr32.exe /s "$(ProjectDir)$(OutDir)$(TargetName)-AddIn.xll"
-</PostBuildEvent>
+regsvr32.exe /s "$(ProjectDir)$(OutDir)$(TargetName)-AddIn.xll"</PostBuildEvent>
   </PropertyGroup>
-  <Import Project="packages\ExcelDna.AddIn.1.1.1\build\ExcelDna.AddIn.targets" Condition="Exists('packages\ExcelDna.AddIn.1.1.1\build\ExcelDna.AddIn.targets')" />
+  <Import Project="packages\ExcelDna.AddIn.1.5.1\build\ExcelDna.AddIn.targets" Condition="Exists('packages\ExcelDna.AddIn.1.5.1\build\ExcelDna.AddIn.targets')" />
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+      <ErrorText>这台计算机上缺少此项目引用的 NuGet 程序包。使用“NuGet 程序包还原”可下载这些程序包。有关更多信息，请参见 http://go.microsoft.com/fwlink/?LinkID=322105。缺少的文件是 {0}。</ErrorText>
```

Comment:
Could you perhaps copy the English text back here?

Reply:
Done.